### PR TITLE
Fixes Release Notes URLS

### PR DESCRIPTION
**Pull request by javierpardollama**

### CODE REVIEW

**Review by joseluiseiguren — APPROVED** [review]

File: NuGet/Cotecna.Domain.Core/Cotecna.Domain.Core.csproj [hunk 1]
```diff
@@ -10,12 +10,12 @@
...
-		<Version>1.4.0</Version>
-		<FileVersion>1.4.0</FileVersion>
-		<AssemblyVersion>1.4.0</AssemblyVersion>
+		<Version>1.4.1</Version>
+		<FileVersion>1.4.1</FileVersion>
+		<AssemblyVersion>1.4.1</AssemblyVersion>
 		<PackageLicenseFile>LICENSE</PackageLicenseFile>
 		<PackageProjectUrl>https://github.com/Cotecna-Inspection/Domain.Core</PackageProjectUrl>
-		<PackageReleaseNotes>https://github.com/Cotecna-Inspection/Domain.Core/tree/main/NuGet/Cotecna.Domain.Core/blob/main/changelog.md</PackageReleaseNotes>
+		<PackageReleaseNotes>https://github.com/Cotecna-Inspection/Domain.Core/blob/main/NuGet/Cotecna.Domain.Core.Configuration/changelog.md</PackageReleaseNotes>
```
Comment (Xelit3) [inline NuGet/Cotecna.Domain.Core/Cotecna.Domain.Core.csproj:18]: now it's pointint to tha change of configuration

Comment (Xelit3) [reply NuGet/Cotecna.Domain.Core/Cotecna.Domain.Core.csproj:18]: Should be


https://github.com/Cotecna-Inspection/Domain.Core/blob/main/NuGet/Cotecna.Domain.Core/changelog.md

**Review by Xelit3 — CHANGES_REQUESTED** [review]

File: NuGet/Cotecna.Domain.Core/Cotecna.Domain.Core.csproj (hunk 1 above)
Comment (javierpardollama) [reply NuGet/Cotecna.Domain.Core/Cotecna.Domain.Core.csproj]: Done!

**Review by eltjo-k — APPROVED** [review]

**Review by Xelit3 — APPROVED** [review]
Ready to be merged, thanks @javierpardollama

**Pull request merged**